spec/spec_helper.rb

```diff
@@ -1,77 +1,18 @@
-require 'puppetlabs_spec_helper/module_spec_helper'
-require 'rspec-puppet-facts'
-
-require 'spec_helper_local' if File.file?(File.join(File.dirname(__FILE__), 'spec_helper_local.rb'))
```

Comment:
Maybe we can introduce something similar into voxpupuli's modulesync config? Our current alternative is to set a big block of code for the `spec_overrides` variable in `.msync.yaml` and it is a bit ugly :disappointed:

Reply:
we already support it https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/spec/spec_helper.rb.erb#L31 :)

```yaml
---
spec/spec_helper.rb:
  spec_overrides: "require 'spec_helper_methods'"
```

we use that in the collectd module (I broke that on master, but you can see it in the git history)